pmd-compat6/src/it/pmd-for-java/exception_ruleset.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" name="Exception throwing ruleset">
+    <description/>
+    <rule name="ExceptionThrowingRule"
+          language="java"
+          class="net.sourceforge.pmd.ExceptionThrowingRule">
+        <description>Use this rule to produce a processing error.</description>
+    </rule>
```

Comment:
With the `error()` function of XPath, we can create a processing error at runtime as a simple XPath rule:

```suggestion
    <rule name="ExceptionThrowingRule"
          language="java"
          class="net.sourceforge.pmd.lang.rule.XPathRule">
        <description>Use this rule to produce a processing error.</description>
        <priority>3</priority>
        <properties>
            <property name="xpath">
                <value><![CDATA[
error()
            ]]></value></property>
        </properties>
    </rule>
```